.github/workflows/checks.yml

```diff
@@ -61,12 +61,19 @@ jobs:
       - name: Setup asdf
         uses: asdf-vm/actions/setup@v2
 
-      - name: Install sncast 0.6.0
+      - name: Install sncast
         run: |
           asdf plugin add starknet-foundry
           asdf install starknet-foundry 0.6.0
           asdf global starknet-foundry 0.6.0
 
+      - name: Install scarb
+        run: |
+          asdf plugin add scarb
+          asdf install scarb 0.4.0
+          asdf install scarb 0.7.0
+          asdf install scarb 2.6.0
```

Comment:
Not sure, but I think the idea was to use [asdf action](https://github.com/asdf-vm/actions) to install the plugins instead of raw commands.

Reply:
There's a little problem with out setup, as we have multiple `.tool-version` files. As far as I know, asdf action uses only one file.

Reply:
as per https://github.com/software-mansion/starknet-jvm/pull/465/files#r1604740615

Let's verify whether the same actions can be stacked and used to install multiple versions of the same tool

Reply:
It turns out that we don't have to stack actions, instead we can simply create `.tool-versions` which gathers all versions and therefore we can comfortably use  `asdf-vm/actions/install@v3`.
This way we can also get rid of commands installing `starknet-foundry`.